src/components/Modals/Send/utils.ts

```diff
@@ -98,15 +98,21 @@ export const estimateFees = ({
 export const handleSend = async ({
   sendInput,
   wallet,
+  checkLedgerAppOpenIfLedgerConnected,
 }: {
   sendInput: SendInput
   wallet: HDWallet
+  checkLedgerAppOpenIfLedgerConnected: (chainId: ChainId) => Promise<void>
 }): Promise<string> => {
-  const supportedEvmChainIds = getSupportedEvmChainIds()
-
   const state = store.getState()
   const asset = selectAssetById(state, sendInput.assetId ?? '')
   if (!asset) return ''
+
+  const chainId = asset.chainId
+  // The double check here and in broadcastTXID isn't a mistake - getAddress() does on-device address derivation
```

Comment:
but... there is no double check there :P

Reply:
Nice catch haha, that was meant to be in `useSendThorTx`'s `executeTransaction`, however, removed to avoid merge conflicts in the next PR in the stack, and knowing we're going to cleanup and consolidate at implementation-level as a follow-up
93bf620d28